feature/login/src/main/java/com/iteneum/login/presentation/LoginView.kt

```diff
@@ -68,41 +76,46 @@ fun LoginView() {
                     end.linkTo(parent.end)
                 },
             label = stringResource(R.string.lv_email),
-            isValid = false,
-            supportTextError = stringResource(R.string.lv_support_text_error),
-            onValueChange = { /*TODO("This will change the value of the email")*/ }
+            placeholder = stringResource(R.string.lv_example_email),
```

Comment:
please add a gray color to the placeholder

Reply:
Hi Meli! , the color of the placeholder is already gray in the component file, should I add it here as well?